Congo_DR/COD_transform_load_dlt.py

```diff
@@ -45,6 +45,8 @@ def boost_silver():
             .when(col("Province").isNotNull(),
                 lower(trim(regexp_replace(col("Province"), "\d+", ""))))
             .otherwise('Other'))
+        .withColumn('adm1_name', 
+            initcap(regexp_replace(col('adm1_name'), '-', ' ')))
```

Comment:
@nihaas21 Do not explicitly define adm1_name as it's a legacy field name. You can see it's alias from `geo1` to save us having to update all the references in viz to use geo1: https://github.com/dime-worldbank/mega-boost/blob/main/cross_country_aggregate_dlt.py#L20 Modify `geo` below instead.

Reply:
Need clarification on the pipeline where adm1_name's changes transform to geo1. In order to change output in adm1_name column, the parameter passed to .with column needs to be adm1_name. Modifying geo1 column when adm1_name is not null does not generate required results.